library_generation/README.md

```diff
@@ -150,8 +156,43 @@ Use `--include_samples` to specify the value.
 Choose the protoc binary type from https://github.com/protocolbuffers/protobuf/releases.
 Default is "linux-x86_64".
 
-## An example to generate a client library
+### enable_postprocessing (optional)
+Whether to enable the post-processing steps (usage of owlbot) in the generation
+of this library
+Default is "true".
+
+### repo_metadata_json_path (optional)
+Custom `.repo-metadata.json` file to be used in the library generation (this is
+consumed by owlbot)
+If not provided, it will default to the one contained in `repository_path`
+
+### owlbot_sha (optional)
+Owlbot image sha to be used in post-processing
+If not provided, it will default to the one contained in `repository_path`
+
+### repository_path (optional)
```

Comment:
Do you think it's possible to only have `destination_path`, rather than having two parameters having similar meaning?

After enable post processing, we can create a temp folder to store the pre-processed library. In order to use owlbot cli, the temp folder needs to satisfy a path prefix, which can be inferred from `proto_path`.

Reply:
The only interesting situation is with `apigee-connect`. We have the [monorepo-folder](https://github.com/googleapis/google-cloud-java/tree/main/java-apigee-connect) called `apigee-connect`, whereas the [googleapis-gen folder](https://github.com/googleapis/googleapis-gen/tree/master/google/cloud/apigeeconnect/v1/google-cloud-apigeeconnect-v1-java) is `apigeeconnect`. 

This could be solved by inspecting the `repo_short` entry in .repo-metadata.json

Reply:
On a second note, it is also possible to get the destination path via `googleapis-gen` following `proto_path`. That will lead us to the `destination_path` already set in `proto_path_list` (e.g. `google-cloud-asset-v1-java`).

Reply:
I agree with Joe as well, we should try to only use one path. I still don't quite get the need of an additional repo path, can you please explain more about it?

Reply:
I added this logic: we infer `destination_path` from `googleapis-gen` at the specified `proto_path`